<launch>
  <node name="marker_detection" pkg="task_marker" type="marker_detection" respawn="true"  output="screen">
    <param name="t1min" type="int" value="0" />
    <param name="t1max" type="int" value="218" />
    <param name="t2min" type="int" value="241" />
    <param name="t2max" type="int" value="260" />
    <param name="t3min" type="int" value="248" />
    <param name="t3max" type="int" value="260" />
  </node>
  <node name="marker_server" pkg="task_marker" type="marker_server" respawn="true"  output="screen">
  </node>
  <!-- <node name="marker_client" pkg="task_marker" type="marker_client" respawn="true"  output="screen">
  </node> -->
</launch>
